sharding/contracts/sharding_manager.go

```diff
@@ -176,6 +176,32 @@ func (_SMC *SMCTransactorRaw) Transact(opts *bind.TransactOpts, method string, p
 	return _SMC.Contract.contract.Transact(opts, method, params...)
 }
 
+// CHALLENGEPERIOD is a free data retrieval call binding the contract method 0xc3a079ed.
+//
+// Solidity: function CHALLENGE_PERIOD() constant returns(uint256)
```

Comment:
Comment is still good though - I wonder why abigen does this?

Reply:
Yeah. I think abigen removes `_` by default